internals/notifier.py

```diff
@@ -840,21 +840,22 @@ class OTExtensionApprovedHandler(basehandlers.FlaskHandler):
   def process_post_data(self, **kwargs):
     self.require_task_header()
     feature = self.get_param('feature')
-    if feature is None:
-      self.abort(400, 'No feature provided.')
     gate_id = self.get_param('gate_id')
-    if gate_id is None:
-      self.abort(400, 'Extension gate ID not provided.')
     requester_email = self.get_param('requester_email')
-    if not requester_email:
-      self.abort(400, 'Extension requester\'s email address not provided.')
```

Comment:
These abort statements were unnecessary, since the get_param method contains an abort statement in the event the parameter is not provided